.github/workflows/bench.yml

```diff
@@ -77,11 +77,9 @@ jobs:
           pip3 install jtbl
       - name: Set env
         run: |
-          # Default benchmark settings optimized for Aptos, can be overwritten with `env` input
-          echo "RUST_LOG=debug" | tee -a $GITHUB_ENV
+          # Default benchmark settings optimized for light clients, can be overwritten with `env` input
           echo "RUSTFLAGS='-C target-cpu=native --cfg tokio_unstable -C opt-level=3'" | tee -a $GITHUB_ENV
           echo "SHARD_SIZE=4194304" | tee -a $GITHUB_ENV
-          echo "SHARD_BATCH_SIZE=0" | tee -a $GITHUB_ENV
```

Comment:
After https://github.com/argumentcomputer/sphinx/pull/163 we should be able to re-add `SHARD_BATCH_SIZE=0` everywhere, so we shouldn't need to remove those

We should also be able to add `RECONSTRUCT_COMMITMENTS=false` to the vars list, per discussion on zulip

cc @storojs72 that has been looking into these variables